static/specification.html

```diff
@@ -93,34 +105,220 @@
+        </p>
+        <aside class="example">
+          <pre class="HTML">
+            &lt;link rel="monetization" href="https://example.com/monetization.js"&gt;
+            &lt;script&gt;
+              // Checking via DOMTokenList
+              const link = document.querySelector('link[rel="monetization"]') ||
+                document.createElement("link");
+              if (link.relList.supports("monetization")) {
+                console.log("Web Monetization is supported.");
+              }
+
+              // Checking global scope for MonetizationEvent
+              if (window.MonetizationEvent) {
+                console.log("Web Monetization is supported.");
+              }
+
+              // Checking if it's a global event handler
+              if (window.hasOwnProperty("onmonetization")) {
+                console.log("Web Monetization is supported.");
+              }
+            &lt;/script&gt;
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Monetizing a web page
+        </h3>
+        <aside class="example">
+          <pre class="html">
+            &lt;!DOCTYPE html&gt;
+            &lt;html lang="en"&gt;
+            &lt;meta charset="utf-8"&gt;
+            &lt;title&gt;My Blog&lt;/title&gt;
+            &lt;link
+              rel="monetization"
+              href="https://example.com/pay"
+              onmonetization="sayThanks(this)"&gt;
+
+            &lt;script&gt;
+              function sayThanks(monetizedLink) {
+                // Do something here
+              }
+            &lt;/script&gt;
+          </pre>
+        </aside>
+      </section>
+      <section>
+        <h3>
+          Monetization events
+        </h3>
+        <p>
+          The {{MonetizationEvent}} DOM events provide information such as the
+          amount sent, the currency of the payment (what we call the "asset
+          code"), and a link to the incoming payment at the [=monetization
+          receiver=] which can be used to verify the receipt of payment. As in
+          the previous example above, these events are dispatched to [^link^]
+          elements and bubble up the DOM tree.
+        </p>
+        <p>
+          To listen for {{MonetizationEvent}} events, an [=event listener=]
+          needs to be added to the relevant [^link^] element (or to one of its
+          ancestors) via JavaScript:
+        </p>
+        <aside class="example">
+          <pre class="html">
+            &lt;link rel="monetization" href="https://example.com/pay"&gt;
+            &lt;script&gt;
+              const link = document.querySelector('link[rel="monetization"]');
+              link.addEventListener("monetization", event =&gt; {
+                // See how much was sent and in what currency
+                const { amount, assetCode, assetScale } = event;
+                console.log(`Browser sent ${assetCode}${amount / (10 * assetScale)}.`);
```

Comment:
> We are also training people with these examples to use the "string" typed amount as a number,

From https://webmonetization.org/docs/counter:
> Even though ev.detail.amount is a string that can represent a number up to 64 bits long in theory (too big for a JavaScript number), it's OK for us to convert it to a JavaScript number here. The micropayment amounts in Web Monetization are far from pushing the limits. With USD and a scale of 9, for instance, it would take around 500,000 years to overflow this number.

I believe STREAM [encodes](https://github.com/interledger/rfcs/blob/master/asn1/Stream.asn) amounts as varInt. 
TigerBeetle encodes transaction amounts as Uint64. 

If the packets are in practice small enough to be "OK", perhaps that should be enforced somewhere?